101-workshop-nodejs/ex0-what-is-serverless/serverless-programming-model.md

```diff
@@ -17,108 +17,106 @@
+The conceptual list of automated sources that can be processed by serverless functions is endless!
 {% endhint %}
 
-## The **Feed** is the event adapter
-
-In the case of ICF's model, the _Event Source_ is more conceptual and the **Feed** actually represents an "adapter" service in the system that understands how to connect to and/or receive data from an _Event Source_, adapt it to a normalized form and then "feed" it to one or more functions by invoking _Triggers_.
+## The Feed is the event adapter
 
-### Feeds follow one of three patterns:
+In the case of the ICF's model, the event source is more conceptual and the **Feed** represents an adapter service in the system. The feed understands how to connect to and/or receive data from an event source, adapt it to a normalized form, and then feed it to one or more functions by invoking triggers. Feeds follow one of three patterns:
 
-#### _Hook_
+1. Hook: A feed uses a webhook facility or callback mechanism exposed by an external service that generates events like an event source.
 
-A feed uses a "webhook" facility or "callback" mechanism exposed by an external  service that generates events (i.e., an event source).
+1. Polling: A service that polls an external service endpoint periodically to fetch new data and generate its own events.
 
-#### _Polling_
+1. Connection: A dedicated running service that maintains a persistent connection to an event source (for example, implementing a client of a message queue service or database) that creates and generates events on its behalf.
 
-A service that polls an external service endpoint periodically to fetch new data and generate its own "events".
-
-#### _Connection_
-
-A dedicated service running somewhere that maintains a persistent connection to an event source (e.g., implements a client of a message queue service or database) creates events. These types of feed implementations are called event **Provider Services**.
+Feeds that are implemnented as long-running services are sometimes referenced as event provider services.
 
 {% hint style="tip" %}
-Later in the course, we will show how to actually implement a **Polling Feed Service** using a Serverless function that is periodically triggered from an  Alarm!
+Later in the course, you will learn how to implement a **polling feed service** using a serverless function that is periodically triggered from an  alarm!
 {% endhint %}
 
-## Why does ICF use **Triggers**?
+## Why does ICF use triggers?
 
-Triggers are not part of every Serverless programming model, but are a powerful concept within IBM Cloud Functions that supports the [Observer design pattern](https://en.wikipedia.org/wiki/Observer_pattern) effectively.
+**Triggers** are not part of every serverless programming model but are a powerful concept within ICF that supports the [observer design pattern](https://en.wikipedia.org/wiki/Observer_pattern) effectively.
```

Comment:
Please leave emphasis on Observers and preserve "sinks" as a programming term.

Reply:
If you'd like to keep oberevers italicized, that's fine. According to the wikipedia documentation, it doesn't require capitalization. Sinks in quotations can be left as is.